configuration/product_variant/products.csv

```diff
@@ -0,0 +1,38 @@
+id,name,standard_price,lst_price,uom_id/id,uuid,product_variant_ids/categ_id/id,type,uom_po_id/id

```

Comment:
Rename the headers as per the specs:
`product_variant_ids/categ_id/id`

Reply:
Done